mongodbatlas/fw_provider.go

```diff
@@ -0,0 +1,394 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/int64validator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/listvalidator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/provider"
+	"github.com/hashicorp/terraform-plugin-framework/provider/schema"
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-mux/tf5to6server"
+	"github.com/hashicorp/terraform-plugin-mux/tf6muxserver"
+	sdkv2schema "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	cstmvalidator "github.com/mongodb/terraform-provider-mongodbatlas/mongodbatlas/framework/validator"
+)
+
+type MongodbtlasProvider struct{}
+
+type tfMongodbAtlasProviderModel struct { //nolint:govet //prefer to maintain order of attributes over small byte optimization
```

Comment:
```suggestion
type tfMongodbAtlasProviderModel struct { //nolint:govet //prefer to maintain order of attributes over small byte optimization
```

this is not what we decided to do for TF. happy to open again this discussion if you want (feel free to open a discussion on team slack channel) but for now, you should run `make fix-lint` and remove `nolint`

Reply:
Adjusted. No strong feeling here, just found it less error prone to preserve a certain order during development.